src/core/libmaven/libmaven.pro

```diff
@@ -13,8 +13,7 @@ QMAKE_CXXFLAGS +=  -std=c++11
 QMAKE_CXXFLAGS += -DOMP_PARALLEL
 linux: QMAKE_CXXFLAGS += -Ofast -ffast-math
 win32: QMAKE_CXXFLAGS += -Ofast -ffast-math
-!macx: QMAKE_CXXFLAGS += -fopenmp
-!macx: LIBS += -fopenmp
```

Comment:
Adding openMP to LIBS for all platform here but not in mzroll. Why is that? just want to understand

Reply:
LIBS flag is used by the linker and not by the compiler, and all the linker flags have to be added in the pro files responsible for creating the binary(mzroll.pro, mavenTests.pro, peakdetector.pro). Hence, this flag is not needed here. 

Hope I answered your question

Reply:
Got it. Thanks!